tmt/base.py

```diff
@@ -4670,6 +4670,13 @@ def go(self) -> None:
 
         self.show_runner(self._logger)
 
+        # Create scripts directory and copy tmt scripts there
+        tmt_parent = Path(tmt.__file__).resolve().parent
+        source = tmt_parent / "steps" / "scripts"
+        destination = self.workdir / "scripts"
+        destination.mkdir(exist_ok=True)
+        shutil.copytree(source, destination, dirs_exist_ok=True)
+
```

Comment:
Let's create a new method which would handle copying scripts to make the `go()` method more readable/maintainable.